src/components/entity/ha-entity-picker.ts

```diff
@@ -61,6 +61,8 @@ class HaEntityPicker extends LitElement {
   @property() public label?: string;
   @property() public value?: string;
   @property({ attribute: "domain-filter" }) public domainFilter?: string;
+  @property({ type: Array, attribute: "exclude-domains" })
+  public excludeDomains?: string[];
```

Comment:
How is this different than domain-filter ? We shouldn't have two attributes that do the same, but slightly different things. Let's add some docs above each property.